README.md

```diff
@@ -1,37 +1,33 @@
-
-Right now, the flow of data is `notes frontend <- notes backend <- fs library <- ipfs`.
-
-## Building & Running
-
-To rebuild the notes app, run `make install-notes` from the root directory. This will rebuild both the frontend and backend, and then install them in the right directories in `procs`pp, run `make install-notes` from the root directory. This will rebuild both the frontend and backend, and then install them in the right directories in `procs`.
-
-To start the Habitat service, run `make run-dev` in the root directory of this repository. Habitat will start listening for commands from `habitatctl`. `make run-dev` will rebuild the `habitat` and `habitatctl` binaries.
-
-Right now, the two commands `habitatctl` supports are `habitatctl start` and `habitatctl stop` which are used to start and stop apps on Habitat. For example, to start the notes app, you would run the following:
+## Building and Running
 
+The two main binaries that can be built from this repository are `habitat` and `habitatctl`. The `habitat` program is a server, and `habitatctl` is a command line client for interacting with it. To build these, run
 ```
-habitatctl start ipfs
-habitatctl start notes_backend
-habitatctl start nginx
+make -C cmd build
 ```
+For local development, these binaries are placed in the `./dist/bin` folder relative to the project directory. To run the `habitat` server in development mode, run:
+```
+make run
+```
+Then, you can use the `habitatctl` client to interact with the server. For example:
+```
+./dist/bin/habitatctl inspect
+```
+This repository also contains a number of application projects, which are stored under the `./apps` directory. To build and install them for local development, run:
+```
+make -C apps all install
+```
+This will build all application binaries, web files, etc necesarry for `habitat` to start these apps as subprocesses. 
 
-### Development on notes app
-
-For development on the Notes app, its usually better to just have IPFS running via `habitatctl start`, but then running the frontend and backend on your machine.
-
-Frontend:
-Run `npm start` in the frontend directory. This will run the frontend with hot reloading.
-
-Backend:
-Right now, just `./out/backend/notes-api` from the notes app directory. We should find a better way of running this.
+## Source Directories
+We do our best to follow the [standard Go project layout](https://github.com/golang-standards/project-layout). This is a work in progress, and many things should be moved into an `internal` folder that does not yet exist.
 
-### Running community inside Docker Container:
+## Runtime Directories
+All files generated at runtime are placed under the `HABITAT_PATH` directory. This includes all object store, Raft log stable storage, and node configuration. In development, `HABITAT_PATH` is set to the `.habitat` direectory under the root project directory. On a typical deployed instance of Habitat, it might be found at `~/.habitat` for the user running it.
```

Comment:
should we add documentation about the subdirectories under it as well?

Reply:
I'm gonna keep it as it is now in case things change in the future. The current description covers most of what we have in there